README.md

```diff
@@ -291,3 +291,35 @@ Lots of method/type names have changed between v2.0 and v3.0, but there's very l
 
 * Unit Tests
 * Add support for Builders for objects with no default constructor.
+
+## Creating a new Nuget Package (Dev Notes)
+
+With .Net Standard, you no longer have to use a nuspec file since all the package information is added to the csproj file. You will, however, need VS >= 2017
+
+Process for releasing a new NugetPackage:
+
+* Update the `LochNessBuilder.csproj` with any new information. This can also be done in the package tab of the project properties:
+  * Identify the appropriate new semantic version number. See here if you're unsure: <https://semver.org/>
+  * Update the version numbers. Note that `PackageVersion` needs to be manually updated in the `.csproj` file not the VS UI, as the VS UI seems to be buggy.
+  * Write some appropriate PackageReleaseNotes in the package data.
+* Update the READMEs
+  * Update the `.\README.md` file in general, and if appropriate add a "Migration" section to it.
+  * Update the `.\nugetREADME.md` file, with an appropriate selection from them main `README.md` file.
+* Create the new package.
+  * Clean and Rebuild the solution, which will automatically package everything up for you.
+    * The newly created package will be dropped in the `<root>\nuget` folder.
+  * Delete the previous version which will be sitting beside the newly produced version.
+    * Note that a clean copy of the previous version will already have been kept, in `<root>\nuget\ArchivedPackages`, when it was released. Double check that it's there and if not, look through git to recreate it.
+* Deploy to Nuget!
+  * Log into nuget with an account that is part of the LochNess organisation (Fet an existing member of the org to add you if not already included)
```

Comment:
Worth a link to the [LochNess Organisation](https://www.nuget.org/organization/LochNess)?

Fet > Get. Also, worth mentioning the names and usernames of the current LochNess organisation admins? And as a fallback, the email address for the LochNess Org routes to Softwire Helpdesk.

Reply:
Done.